### ISSUE TITLE
Cannot get it to work.

### ISSUE THREAD

**Issue by R18B0T** [DrNoLife/Destiny-2-Solo-Enabler#41]
I have tried following the FAQ. installing the newest version of Net framework, making sure the windows firewall was the only one enabled, ect.  But I cannot seem to get it to work.  I would love to play D2 but if I am going to be forced to play with others im just not going to.

**Comment by DrNoLife** [comment]
I just went ahead and tested it for myself. I can confirm it still works.

You need to make sure that you are using the Windows Firewall, and nothing else. A lot of extra anti-virus programs also include their own firewall. If you are using any of those, the program will not work.

Make sure you haven't disabled the firewall: https://github.com/DrNoLife/Destiny-2-Solo-Enabler/issues/40
Really, make sure you haven't disabled the firewall: https://github.com/DrNoLife/Destiny-2-Solo-Enabler/issues/29#issuecomment-1368291938
If you are using the proper firewall, but are still encountering issues, then try to delete the default Destiny 2 firewall rules when using the program (the latest version of the program has a setting to automatically do this) https://github.com/DrNoLife/Destiny-2-Solo-Enabler/issues/28#issuecomment-1236405479

**Comment by R18B0T** [comment]
I'm not sure what was going on.  I had all that said and done. even restarted my pc a couple of times. I booted up my PC This morning and now it's working. I haven't done anything differently. maybe Symantec was still doing stuff despite me turning it off several times. either way sorry for bothering you.

**Comment by R18B0T** [comment]
so it was working. Took forever to find a match, joined in, played by myself, then someone randomly someone joined me mid-strike.

**Comment by R18B0T** [comment]
![image](https://user-images.githubusercontent.com/128274581/226190374-1ebc8c86-b399-4d39-b272-5f6320916193.png)
![image](https://user-images.githubusercontent.com/128274581/226190385-acf84070-0792-415d-8b77-9eca36392995.png)
And im trying to run exodus crash. no reason in particular for that strike it was just the first I clicked on. 

also, I don't think it would cause any problems but I do have another anti virus installed however I have it completely turned off while I'm trying to use this.
![image](https://user-images.githubusercontent.com/128274581/226190612-082a020f-92b3-407e-b0d8-6cc8de24a3ed.png)

**Comment by R18B0T** [comment]
![image](https://user-images.githubusercontent.com/128274581/226191426-386776e2-c073-46a5-a9f4-9ae3c60ce9bb.png)
it just stops working after a while

**Comment by R18B0T** [comment]
and before its brought up yes its a different strike i was seeing if for some reason it was exclusive to one strike

**Comment by BLTplayz** [comment]
I believe that the issue is still related to your antivirus. Even though it is disabled, it may still be doing some routing separate from the windows firewall. 

I am not sure how AVs handle firewalls but I think you should try looking through the rules to see if there is a rule that just forwards all ports. I think the antivirus would add a rule like this to forward all traffic to its firewall and it may not be properly disabled when you turn off the AV. If there is one, disable it and test again. 

If there is no such rule or it still doesn't work, make sure the AV program itself is not running **at all.** Check the task manager for any rogue programs related to your AV and stop them. If it still fails, I'd uninstall the AV and test again. (Having an AV really complicates the internal network routing so if this did fix it, I'd cut my losses and give up on using SoloEnabler as it is better to have the extra security of an AV)

Finally, just try adding your own firewall rule to block ports 27000-27200, 3097 on both TCP and UDP, inbound and outbound. This is just a bandaid fix as you'd have to manually enable and disable the rules every time you wanted to use them. Also, this is really tedious to do so I understand if you don't want to try this haha. But if making the rules does not fix it, I cannot think of any other solutions to this issue :(

But do reply and let me know if you've figured something out, Id love to know more about this specific case.

**Comment by Geist4587** [comment]
Please please watch my videos I don't want any followers I need the help of the masses Bungie the Destiny 2 developers have been hacking my HP Omen PC for over a year I've asked for help everywhere Facebook Instagram YouTube and Twitter put me in incognito so the truth  It doesn't come out, the sad thing is I haven't even paid it off completely and they don't stop hacking me, please distribute my videos, help me, I've been fighting Bungie for a long time with depression.  https://youtube.com/shorts/feRDkmBOyA4?si=GoWKFnyxqIi-eWkg

**Comment by Cr3aHal0** [comment]
Hi everyone

Just posting this message because I was having a hard time making the script work for 2 months (it didn't work). Windows firewall setup was correct and activated but a Kaspersky upgrade occured on my side  3 months ago. I did desactivate Kaspersky integrated Firewall and it then worked well !

TLDR : Check for antivirus Firewall :)

Sorry for my approximative english, hope this helped tho
